src/index.html

```diff
@@ -5,8 +5,412 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#" class="header__logo-link">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-list-item-1">
+            <a
+              href="#"
+              class="header__nav-list-item-1-link is-active"
+              data-qa="is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="header__nav-list-item-2">
+            <a href="#" class="header__nav-list-item-2-link">
+              samsung
+            </a>
+          </li>
+          <li class="header__nav-list-item-3">
+            <a href="#" class="header__nav-list-item-3-link">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="header__nav-list-item-4">
+            <a
+              href="#"
+              class="header__nav-list-item-4-link"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="header__nav-list-item-5">
+            <a href="#" class="header__nav-list-item-5-link">
+              гаджеты
+            </a>
+          </li>
+          <li class="header__nav-list-item-6">
+            <a href="#" class="header__nav-list-item-6-link">
+              планшеты
+            </a>
+          </li>
+          <li class="header__nav-list-item-7">
+            <a href="#" class="header__nav-list-item-7-link">
+              фото
+            </a>
+          </li>
+          <li class="header__nav-list-item-8">
+            <a href="#" class="header__nav-list-item-8-link">
+              видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__catalog">
+        <div class="main__catalog-product-1">
+          <div class="main__catalog-product-1-card" data-qa="card">
+            <img
+              src="images/imac.jpeg"
+              alt="imac"
+              class="main__catalog-product-1-imac-photo"
+            >
+            <span class="main__catalog-product-1-name">
+              Моноблок APPLE A1419 iMac<br> 27" Retina 5K (MNED2UA/A)
+            </span>
```

Comment:
rename all elements to be a card's children, also simplify the namings, they're too long

```suggestion
            <img
              src="images/imac.jpeg"
              alt="imac"
              class="card__image"
            >
            <span class="card__name">
              Моноблок APPLE A1419 iMac<br> 27" Retina 5K (MNED2UA/A)
            </span>
```